src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,174 @@
+package Duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+public class Duke {
+
+    public static void main(String[] args) throws IOException {
+        welcomeMessage();
+
+        ArrayList<Task> taskList = new ArrayList<>();
+        int taskCount = 0;
+
+        File dataDir = new File("data");
+        dataDir.mkdir();
+        File dataFile = new File("data/tasks.txt");
+        if (!dataFile.createNewFile()) {
+            taskCount = downloadTasks(dataFile, taskList, taskCount);
+        }
+
+        Scanner in = new Scanner(System.in);
+        String commandInput = in.nextLine();
+
+        while (!commandInput.equals("bye")) {
+            //Process commands taken from user, exit if user typed "bye"
+            if (commandInput.equals("list")) {
```

Comment:
Try to avoid multiple if-else statements. Can consider using switch statements instead